code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -174,6 +174,7 @@
 			var/new_nutrition = round(nutrition * 0.9)
 			var/new_powerlevel = round(powerlevel / 4)
 			var/datum/component/nanites/original_nanites = GetComponent(/datum/component/nanites)
+			var/turf/drop_loc = get_turf(src)
```

Comment:
This should still let slimes reproduce inside valid containers. Lockers shouldn't cause issues, for example, although it could get cramped
```suggestion
			var/turf/drop_loc = drop_location()
```